mbed_app.json

```diff
@@ -30,6 +30,23 @@
             "nsapi.default-wifi-security": "WPA_WPA2",
             "nsapi.default-wifi-ssid": "\"SSID\"",
             "nsapi.default-wifi-password": "\"PASSWORD\""
+        },
+        "EP_AGORA": {
```

Comment:
Does the example pick up cellular by default on this target? I mean the interface returned by `NetworkInterface::get_default_instance()` in `main.cpp`.

Reply:
If we are not certain, we can add
```
"target.network-default-interface-type": "CELLULAR",
```

Reply:
It is the default for this target, see https://github.com/ARMmbed/mbed-os/blob/master/targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_EP_AGORA/mbed_lib.json/#L17